zulipterminal/ui.py

```diff
@@ -176,16 +181,16 @@ def main_window(self) -> Any:
         )
         return w
 
-    def show_left_panel(self, *, visible: bool) -> None:
-        if not self.autohide:
+    def show_left_panel(self, *, visible: bool, ignore_check: bool = False) -> None:
+        if not self.autohide and self.mode != "small" and not ignore_check:
```

Comment:
Can we use `visible` instead of `ignore_check`?

Reply:
We might want to hide (visible=False) but skip the check. That would not be possible right?